conf/messages.cy

```diff
@@ -3463,7 +3463,7 @@ repayment.personalRepInPeriodOfAdmin.agent.title=Yn seiliedig ar y manylion yn y
 
 repayment.yes.content=Os gwnaethoch dalu â cherdyn debyd, byddwn yn ceisio eich ad-dalu. Os gwnaethoch dalu drwy ddull arall, gofynnir am eich manylion banc ar dudalen arall yn nes ymlaen.
 
-repayment.helpText=Os dewiswch ‘Iawn’ a bod ad-daliad Treth Enillion Cyfalaf yn ddyledus i chi, bydd CThEM yn ad-dalu’r arian i chi cyn pen 30 diwrnod. Os nad oes ad-daliad yn ddyledus i chi, ni chysylltir â chi.
+repayment.helpText=Os dewiswch ‘Iawn’ a bod ad-daliad Treth Enillion Cyfalaf yn ddyledus i chi, bydd CThEM yn ad-dalu’r arian i chi. Os nad oes ad-daliad yn ddyledus i chi, ni chysylltir â chi.
 repayment.agent.helpText=Os dewiswch ‘Iawn’ a bod ad-daliad Treth Enillion Cyfalaf yn ddyledus i’ch cleient, bydd CThEM yn ad-dalu’r arian iddo cyn pen 30 diwrnod. Os nad oes ad-daliad yn ddyledus iddo, ni chysylltir ag ef.
 repayment.trust.helpText=Os dewiswch ‘Iawn’ a bod ad-daliad Treth Enillion Cyfalaf yn ddyledus i’r ymddiriedolaeth, bydd CThEM yn ad-dalu’r arian cyn pen 30 diwrnod. Os nad oes ad-daliad yn ddyledus iddi, ni chysylltir â chi.
 repayment.capacitor.helpText=Os dewiswch ‘Iawn’ a bod ad-daliad Treth Enillion Cyfalaf yn ddyledus i’r person, bydd CThEM yn ad-dalu’r arian iddo cyn pen 30 diwrnod. Os nad oes ad-daliad yn ddyledus iddo, ni chysylltir ag ef.
```

Comment:
Like english version, you need to update this message for all users like agent, capacitor, personalRep, personalRepInPeriodOfAdmin,trust too.